src/web/src/pages/CoursePage.vue

```diff
@@ -117,5 +117,18 @@ export default {
       return precoreqtext;
     },
   },
+  metaInfo() {
+    return {
+      title: this.courseTitle,
+      titleTemplate: "%s | YACS",
+      meta: [
+        { charset: 'utf-8' },
+        { name: 'description', content: this.courseObj.description},
+        { property: 'og:title', content: "YACS - RPI Course Scheduler"},
```

Comment:
Also, maybe lets switch this to RPI - YACS Course Scheduler, we'll parametrize this a bit later in admin panel